doc/specs/top_10.md

```diff
@@ -0,0 +1,147 @@
+# Charts
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+Users can view the most popular items on the website based on various filters in terms of content type and date range. Users will be able to view charts in a section on their homepage or on a separate dedicated page.
+
+## Business rules
+
+- Users can enable/disable the charts of their choice from displaying on their homepage in batches of 10.
+- Users can view all charts on the dedicated charts page in batches of 10, 100, or 250 items at a time.
+- Users can view all charts by past day, past week, past month, or all-time.
```

Comment:
Charts will be of the following categories: most active, most snatched, most data transferred, and best seeded.
Only the "most active" category will be broken down into past day/week/month/all-time.

That would be if we recreated the existing thing. Maybe there is no real need for all of them, though.